### PR TITLE
[Docs] update installation guides

### LINKED ISSUE #1918: [Docs] update installation guides
**Current State:**

The installation guides don't mention `mesheryctl` as a prerequisite to installing Meshery on any supported platform.

**Desired State:**

Add a disclaimer  to the installation guides mentioning `mesheryct` as a prerequisite

---
**Contributor Resources**
- [Meshery documentation site](https://meshery.layer5.io/docs/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by chandrashritii**
**Description**

- add mesheryctl as a prerequisite
- add platform icons
- fix broken links
- styling changes

This PR fixes #1918 

Signed-off-by: Shriti <shritichan@gmail.com>

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/pages/installation/aks.md [hunk 1]
```diff
@@ -7,20 +7,35 @@ language: en
 list: include
 ---
 
-## Connect Meshery to Azure Kubernetes Cluster
+## Quick start with Azure Kubernetes Service (AKS) <img src="/docs/assets/img/platforms/aks.svg" width="35" height="35" />
+
+###### **Prerequisite(s)**
+
+- Install [mesheryctl]({{ site.baseurl }}/guides/mesheryctl)
```
Comment (leecalcote) [inline docs/pages/installation/aks.md:14]: @chandrashritii will you move this to a common include file?

**Review by leecalcote — CHANGES_REQUESTED** [review]
@kushthedude how does the rest look?

File: docs/pages/installation/eks.md [hunk 2]
```diff
@@ -7,7 +7,13 @@ language: en
 list: include
 ---
 
-## Quick Start with Amazon Elastic Kubernetes Service (EKS)
+## Quick Start with Amazon Elastic Kubernetes Service (EKS) <img src="/docs/assets/img/platforms/eks.png" width="35" height="35" />
```
Comment (kushthedude) [inline docs/pages/installation/eks.md:10]: This section needs to be removed and replace with coming soon, @leecalcote

**Review by leecalcote — DISMISSED** [review]

**Review by leecalcote — CHANGES_REQUESTED** [review]
@chandrashritii please address conflicts.

**Review by leecalcote — COMMENTED** [review]
Closing this PR the same updates were included across multiple PRs.

**Pull request closed without merging**